packages/electrode-archetype-react-app-dev/config/webpack/partial/test-base.js

```diff
@@ -14,7 +14,17 @@ module.exports = {
     noParse: [new RegExp(Path.normalize("node_modules/sinon/").replace(/\\/g, "\\\\"))]
   },
   devServer: {
-    stats: "errors-only" // only show errors
+    stats: "errors-only", // only show errors
+    /*
+     * Karma browsers and webpack are running on different ports, which introduces
+     * CORS issue.
+     * The following code allows karma launched browsers to access webpack
+     * bundle in `test-watch` and `test-watch-all` modes.
+     */
+    disableHostCheck: true,
```

Comment:
I slightly doubt that this is the right place for this configuration. If there is a better idea - let me know